benchmarks/benchmarks/bench_avx.py

```diff
@@ -39,6 +39,19 @@ def setup(self, ufuncname, stride, dtype):
     def time_ufunc(self, ufuncname, stride, dtype):
         self.f(self.arr[::stride])
 
+class AVX_UFunc_log(Benchmark):
```

Comment:
`np.log` already has a benchmark test in `AVX_UFunc`, Is there any special reason to write a new one?

Reply:
Yes, the old benchmark using a ones-array as input. This code will call glibc's log function when input equals to 1. Old benchmark can not show out the enhancement of performance. And, glibc implement log in a dispatch when input equals to 1, which is faster than usually.